documents/Log.md

```diff
@@ -0,0 +1,35 @@
+# Development Log
```

Comment:
can you make a folder called dev logs? and instead of naming this log.md, can you name it week1Log.md